pkg/kubelet/kubelet_node_status.go

```diff
@@ -57,6 +56,9 @@ const (
 	maxNamesPerImageInNodeStatus = 5
 )
 
+// Last time we patch node status.
+var lastHeartbeatTime = metav1.NewTime(time.Now())
```

Comment:
don't use a package var for this... if we need to track it, do so in the Kubelet struct, and ensure access to it is threadsafe

Reply:
No multi-thread access now. move to  Kubelet struct sounds ok.